src/validation.cpp

```diff
@@ -602,10 +602,8 @@ static bool AcceptToMemoryPoolWorker(const CChainParams& chainparams, CTxMemPool
     std::set<uint256> setConflicts;
     for (const CTxIn &txin : tx.vin)
     {
-        auto itConflicting = pool.mapNextTx.find(txin.prevout);
-        if (itConflicting != pool.mapNextTx.end())
-        {
-            const CTransaction *ptxConflicting = itConflicting->second;
+        const CTransaction* ptxConflicting = pool.GetConflictTx(txin.prevout);
```

Comment:
Nit: I think the convention is to not prefix variables with types anymore and to use `snake_case`.

Reply:
It is used in a couple of places further down, so I will keep the name for now.

Reply:
Weird, it looked like you renamed `itConflicting` to `ptxConflicting`, but `ptxConflicting` existed already. My bad.